gunicorn_conf.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Gunicorn production config."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from click import echo
+
+bind = '0.0.0.0:5000'
+
+preload_app = True
```

Comment:
Maybe add a comment about why we want this? Same goes for `gthread` below. It's not obvious why these are our desired settings from their name alone.